src/resources/UsersContext.js

```diff
@@ -0,0 +1,48 @@
+import 'isomorphic-fetch';
+import PagedContext from './PagedContext';
+import User from './User';
+
+/**
+ * User querying context
+ *
+ * This is used to query the list of users for a customer
+ */
+class UsersContext extends PagedContext {
```

Comment:
This should probably be called `CustomerUsersContext` (or something similar) to denote that the context applies to users associated to a customer.